docs/Projects/LLM-Benchmarks/index.md

```diff
@@ -0,0 +1,27 @@
+# LLM Benchmarks
+
+This document contains the design decisions behind the LLM Benchmarks the architectural diagrams and the technical
+solution can be found [here](https://github.com/MinBZK/llm-benchmark).
+
+The requirements of the solution are:
+
+- People should be able to run the LLM Benchmark system locally
+    - When the BIO does not allow the model to leave the digital organizational boundary
+    - Locally we have to get the hash of the model as a version stamp
```

Comment:
I think this is a requirement: We should be able to version the model

Reply:
agree